packages/side-nav/test/dom/side-nav-item.test.js

```diff
@@ -0,0 +1,73 @@
+import { expect } from '@esm-bundle/chai';
+import { fixtureSync, nextFrame, nextRender } from '@vaadin/testing-helpers';
+import '../../enable.js';
+import '../../src/vaadin-side-nav-item.js';
+import '@vaadin/icon';
+import '@vaadin/icons';
+
+describe('vaadin-side-nav-item', () => {
+  let sideNavItem;
+
+  beforeEach(async () => {
+    sideNavItem = fixtureSync(
+      `
+        <vaadin-side-nav-item>
+          <vaadin-icon icon="vaadin:chart" slot="prefix"></vaadin-icon>
+          Item
+          <span theme="badge primary" slot="suffix">2</span>
+          <vaadin-side-nav-item slot="children">Child item 1</vaadin-side-nav-item>
+          <vaadin-side-nav-item slot="children">Child item 2</vaadin-side-nav-item>
+        </vaadin-side-nav-item>
+            `,
+    );
+    await nextFrame();
+  });
+
+  describe('item', () => {
+    it('default', async () => {
+      await expect(sideNavItem).dom.to.equalSnapshot();
+    });
+
+    it('expanded', async () => {
+      sideNavItem.expanded = true;
+      await nextRender(sideNavItem);
+      await expect(sideNavItem).dom.to.equalSnapshot();
+    });
+
+    it('active', async () => {
+      const activeItem = fixtureSync('<vaadin-side-nav-item path=""></vaadin-side-nav-item>');
```

Comment:
**nit:** How about this?

```suggestion
      activeItem.path = ''
```

Reply:
I think this would need a path change observer to work.

Reply:
I added an observer and updated the test based on your suggestion.